### ISSUE TITLE
[feat]:  when receiving kubectl command - send immediate response

### ISSUE THREAD

**Issue by ShragaUser** [kubeshop/botkube#571]
**Is your feature request related to a problem? Please describe.**
kubectl commands can take some time, depending on the request they can even takes close to a minute.
we should let the user know that the command was received and is being worked on by the bots backend as soon as possible, and then respond with the answer when its ready.

waiting on a response can sometime be stressful, with the user not being sure if it was even acknowledged by the bot (thinking there might be an error \ issue).

**Comment by pkosiec** [comment]
Hey @ShragaUser, this is already done in one of the recent Botkube versions - Botkube now supports emoji reactions so that you'll know when it picked up your message (#1156) 
Please upgrade your Botkube installation to the latest version 👍 Cheers!
